Modules/ITS/include/ITS/ITSClusterTask.h

```diff
@@ -108,6 +110,8 @@ class ITSClusterTask : public TaskInterface
   TH2F* hAverageClusterOccupancySummaryZPhi[NLayer] = { nullptr };
   TH2F* hAverageClusterSizeSummaryZPhi[NLayer] = { nullptr };
 
+  TH1D* hLaneStatusSummaryGlobal;
```

Comment:
Would it be possible to change name of this histogram to `hEmptyLaneFractionGlobal`? The current name doesn't explain meaning of the object much